ChangeLog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+New in version 3.5.0
+--------------------
+* Fixing a socket closing on ``MemcacheUnexpectedCloseError``
+* Added support of TCP keepalive for client sockets on Linux platforms
+* Added retrying mechanisms by wrapping clients
```

Comment:
This hasn't been merged yet.

Reply:
Yes I know this is why I added a depends-on warning in the description of this pull request